dynamic-simulation/dynamic-simulation-api/src/main/java/com/powsybl/dynamicsimulation/DynamicSimulation.java

```diff
@@ -33,13 +34,13 @@ public Runner(DynamicSimulationProvider provider) {
         }
 
         public CompletableFuture<DynamicSimulationResult> runAsync(Network network, DynamicModelsSupplier dynamicModelsSupplier, EventModelsSupplier eventModelsSupplier, CurvesSupplier curvesSupplier, String workingVariantId,
-                                                                   ComputationManager computationManager, DynamicSimulationParameters parameters) {
-            return provider.run(network, dynamicModelsSupplier, eventModelsSupplier, curvesSupplier, workingVariantId, computationManager, parameters);
+                                                                   ComputationManager computationManager, DynamicSimulationParameters parameters, Reporter reporter) {
```

Comment:
If we keep the same logic as before, we should have kept a 
```
public CompletableFuture<DynamicSimulationResult> runAsync(Network network, DynamicModelsSupplier dynamicModelsSupplier, EventModelsSupplier eventModelsSupplier, CurvesSupplier curvesSupplier, String workingVariantId,
                                                                   ComputationManager computationManager, DynamicSimulationParameters parameters) {
```
which is calling that one with `Reporter.NO_OP`.

Even if it's quite ugly, let's keep the same logic in this PR.